app/CocoaPods.xcodeproj/project.pbxproj

```diff
@@ -729,7 +729,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "if [ \"${CONFIGURATION}\" = \"Release\" ]; then\n\n    LOCATION=\"${BUILT_PRODUCTS_DIR}\"/\"${FRAMEWORKS_FOLDER_PATH}\"\n\n    IDENTITY=\"Developer ID Application: COCODE LTD\"\n\n    codesign --verbose --force --sign \"$IDENTITY\" \"$LOCATION/Sparkle.framework/Versions/A\"\nfi";
+			shellScript = "if [ \"${CONFIGURATION}\" = \"Release\" ]; then\n\n    LOCATION=\"${BUILT_PRODUCTS_DIR}\"/\"${FRAMEWORKS_FOLDER_PATH}\"\n\n    codesign --verbose --force --sign \"$EXPANDED_CODE_SIGN_IDENTITY_NAME\" \"$LOCATION/Sparkle.framework/Versions/A\"\nfi";
```

Comment:
`EXPANDED_CODE_SIGN_IDENTITY_NAME` comes from Xcode, and is whatever signs the `.app`